packages/shorebird_cli/test/src/commands/preview_command_test.dart

```diff
@@ -1267,6 +1278,376 @@ channel: ${DeploymentTrack.staging.channel}
           ).called(1);
         });
       });
+
+      group('when the android release is not sideloadable', () {
+        setUp(() {
+          final unfilteredRelease = MockRelease();
```

Comment:
I don't think it's clear what `unfiltered` means here

Reply:
Renamed to `releatWithAllPlatforms` to match the name using in the command code